src/index.html

```diff
@@ -1,13 +1,306 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Nothing</title>
+    <link
+      rel="icon"
+      href="./image/favicon-32x32.png"
+      type="image/x-icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@700&family=Space+Mono&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="header_logo">
+          <img
+            class="header__img"
+            src="./image/logo.png"
+            alt="nothing_logo"
+          >
+        </a>
+        <div class="header__icons">
+          <a href="tel:+1 234 5555-55-55" class="icon icon--phone">
+            <span class="icon__phone-number">+1 234 5555-55-55</span>
+          </a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+      <div class="header__bottom">
+        <h1 class="header__title">Bring joy back to <br> the everyday</h1>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="top-bar menu__top">
+        <a href="#" class="header_logo">
+          <img
+            class="menu__img"
+            src="./image/logo.png"
+            alt="nothing_logo"
+          >
+        </a>
+        <a href="#" class="icon icon--close"></a>
+      </div>
+
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#products" class="nav__link">Products</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#store" class="nav__link">Store</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#login" class="nav__link">Login</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#cart" class="nav__link">Cart(0)</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+        +1 234 5555-55-55
+        </a>
+        <a href="tel:+1 234 5555-55-55" class="menu__call">
+          call to order
+        </a>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <section class="products" id="products">
+          <h2 class="products__header products--header-margin">Recommended</h2>
+          <div class="products__wrapper">
+            <article class="products__card">
+              <a href="#">
+                <img class="products__image"
+                  src="./image/products/phone.png"
+                  alt="phone"
```

Comment:
![image](https://github.com/mate-academy/layout_landing-page/assets/54860641/741ec1fa-5208-4465-90ef-f27e8624bb87)
Fix this in all code.